engineering-plans/0003-remove-jsonb-storage.md

```diff
@@ -0,0 +1,125 @@
+# PLAN-0001: Remove JSONB Storage
+
+<dl>
+  <dt>Author</dt>
+  <dd>David Lutterkort</dd>
+
+  <dt>Implements</dt>
+  <dd>No RFC - no user visible changes</dd>
+
+  <dt>Engineering Plan pull request</dt>
+  <dd><a href="https://github.com/graphprotocol/rfcs/pull/7">https://github.com/graphprotocol/rfcs/pull/7</a></dd>
+
+  <dt>Date of submission</dt>
+  <dd>YYYY-MM-DD</dd>
+
+  <dt>Date of approval</dt>
+  <dd>YYYY-MM-DD</dd>
+
+  <dt>Approval needed</dt>
+  <dd>Yaniv/Jess/Eva on general approach and help with notifying users</dd>
+  <dd>Core devs on migration strategy</dd>
```

Comment:
Heh, this is not how `dl`/`dt`/`dd` work. You can only have one `dd` element after a `dt`. `dl` is like a glossary where you have pairs of definition topics (`dt`) and definition descriptions (`dd`).

Reply:
The W3C thinks otherwise: https://dev.w3.org/html5/html-author/#the-dl-element :smiley:

Reply:
:mindblown:

Reply:
Having written HTML since 2000 or so... how did I not know this?!